tap_postgres/tap.py

```diff
@@ -1,13 +1,23 @@
 """Postgres tap class."""
+from __future__ import annotations
 
+import atexit
+import io
+import signal
 from functools import cached_property
-from typing import List
+from typing import TYPE_CHECKING, Any
 
+import paramiko
 from singer_sdk import SQLTap, Stream
 from singer_sdk import typing as th  # JSON schema typing helpers
+from sqlalchemy.engine.url import make_url
+from sshtunnel import SSHTunnelForwarder
 
 from tap_postgres.client import PostgresConnector, PostgresStream
 
+if TYPE_CHECKING:
+    from sqlalchemy.engine.url import URL
+
 
 class TapPostgres(SQLTap):
     """Postgres tap class."""
```

Comment:
```suggestion
    """Singer tap for Postgres."""
```

and then call `--about --format=markdown` to update the readme

Reply:
@edgarrmondragon  done!